lib/utils/tests-options-validator.js

```diff
@@ -79,13 +99,17 @@ function validatePartitionSplit(partitions, split) {
  * Ensures that there is no partition listed twice
  *
  * @private
- * @param {Array} partitions
+ * @param {Array} arr
+ * @param {String} typeOfValue
  */
-function validatePartitionsUnique(partitions) {
-  partitions = partitions.sort();
-  for (var i = 0; i < partitions.length - 1; i++) {
-    if (partitions[i] === partitions[i + 1]) {
-      throw new Error('You cannot specify the same partition twice.');
+function validateElementsUnique(arr, typeOfValue) {
+  arr = arr.sort();
+  for (var i = 0; i < arr.length - 1; i++) {
+    if (arr[i] === arr[i + 1]) {
+      const errorMsg = 'You cannot specify the same %TYPE_OF_VALUE% value twice. %VALUE% is repeated.'
```

Comment:
Could you use a backtick template string instead of %TYPE_OF_VALUE% and %VALUE% here?